docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-send-request-in-data-exchange-api.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: How to send a request in Dynamic Data API
```

Comment:
Is there any reason we are describing data exchange api using HTTP and not regular API requests?

Reply:
What is `regular API requests`? @andriitserkovnyi